pinconfigs/sam-l21-ab.yml

```diff
@@ -0,0 +1,456 @@
+# Copyright (c) 2022 Argentum Systems Ltd.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Sources:
+# - SAM_L21_Family_DataSheet_DS60001477C (Revision C - 03/2020)
+#
+# Pin codes:
+#
+# - 32      pins: e
+# - 48      pins: g
+# - 64      pins: j
+#
+# SoC Revision (variant) supported:
+#
+# - SAML21
+# - A = Engineering Samples Only
+# - B = Released to Production
+
+model: atmel,sam
+
+family: l21
+
+map: SAM_PINMUX
+
+series: [l21]
+
+revisions:
+  l21: ab
+
```

Comment:
No need to add revisions for this variant.

Reply:
done